mmv1/templates/terraform/examples/logging_linked_dataset.tf.erb

```diff
@@ -0,0 +1,11 @@
+resource "google_logging_project_bucket_config" "<%= ctx[:primary_resource_id] %>" {
+  location         = "global"
+  project          = "<%= ctx[:test_env_vars]['project'] %>"
+  enable_analytics = true
+  bucket_id        = "tftest%{random_suffix}"
+}
+
+resource "google_logging_linked_dataset" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
Could you add two more tests
1. a handwritten update test to ensure updating the description works. This resource can be marked as immutable if no update is supported.
2. a test for `google_logging_linked_dataset` where parent and location are defined.

Reply:
The resource cannot be updated because none of its attributes are modifiable. Marked immutable. 

Added this test, just want to make sure this isn't going to be confusing/problematic for users because `parent` as defined in the [API docs](https://cloud.google.com/logging/docs/reference/v2/rest/v2/locations.buckets.views/create) is the full bucket path whereas in the terraform (as written now) it's project/<project id> (folder/org parents aren't supported yet but will be in the future).

Reply:
As for your comment on the parent/location/bucket divergence from documentation. There is the alternative presentation of only having bucket and/or parent. Location/parent being separate and having intelligence to detect them is intriguing but I do question the usefulness. Ultimately it's up to you how you would like to design the resource. I think an argument can be make for a single reference vs the split you have now.

Reply:
The pattern I'm using is the same as what was done here: https://github.com/GoogleCloudPlatform/magic-modules/pull/7464/files

I initially did have a single "bucket" parameter which was basically the full parent as specified in the API (no location/parent parameters), but someone else suggested I split it out and since other resources seem to do that, I went ahead with that approach.

Reply:
makes sense to me